arq/jobs.py

```diff
@@ -66,6 +67,10 @@ def __init__(
         self._queue_name = _queue_name
         self._deserializer = _deserializer
 
+    async def abort(self, timeout: Optional[float] = None, *, pole_delay: float = 0.5) -> Any:
+        await self._redis.set('%s%s' % (abort_key_prefix, self.job_id), 1, expire=timeout)
```

Comment:
```suggestion
        await self._redis.set(f'{abort_key_prefix}{self.job_id}', b'1', expire=timeout)
```
Please use f-strings or if that's ugly, `.format()`